test/Project.test.js

```diff
@@ -551,4 +551,9 @@ describe("project", function() {
         expect(project.getLocaleInherit("ko-KR")).toBeUndefined();
         expect(project.getLocaleInherit()).toBeUndefined();
     });
+    test("GetProjectType", function() {
+        expect.assertions(1);
+        var project = ProjectFactory('./test/testfiles', {});
+        expect(project.getProjectType().toString()).toBe("web");
+    });
```

Comment:
Maybe add another unit test for a custom project? What happens then?

Reply:
I added another test case.
In webOS, projectType defines projectType something like webos-js, webos-c. the webos-json-resource plugin uses that info in some places. so I thought it would be better to use a method instead of getting information by 'this.project.options.projectType` i.e) https://github.com/iLib-js/ilib-loctool-webos-json-resource/blob/flutter/JSONResourceFile.js#L279